test/integration/openapi2-test.js

```diff
@@ -11,15 +12,19 @@ let output = '';
 function execCommand(options = {}, cb) {
   output = '';
   let finished = false;
-  if (!options.server) { options.server = `http://127.0.0.1:${PORT}`; }
-  if (!options.loglevel) { options.loglevel = 'warning'; }
-  new Dredd(options).run((error) => {
+  const defaultConfig = {
+    server: `http://127.0.0.1:${PORT}`,
+  };
+
+  const dreddOptions = R.mergeDeepRight(defaultConfig, options);
+
+  new Dredd(dreddOptions).run((error) => {
     if (!finished) {
       finished = true;
       if (error ? error.message : undefined) {
         output += error.message;
       }
-      cb();
+      cb(error);
```

Comment:
I don't think this is wanted, because this will halt the tests. We might have tests inspecting the error as a valid outcome of the test. (Perhaps doesn't matter here in particular, but a similar idiom is on many other places where Dredd is called in tests).